crates/oxc_linter/src/rules/typescript/no_var_requires.rs

```diff
@@ -0,0 +1,107 @@
+use oxc_ast::AstKind;
+use oxc_diagnostics::{
+    miette::{self, Diagnostic},
+    thiserror::Error,
+};
+use oxc_macros::declare_oxc_lint;
+use oxc_span::Span;
+
+use crate::{context::LintContext, rule::Rule, AstNode};
+
+#[derive(Debug, Error, Diagnostic)]
+#[error(
+    "typescript-eslint(no-var-requires): Require statement not part of import statement."
```

Comment:
This error message is so vague 🤔 we have the chance to improve this.

Reply:
I changed the message to be more specific. Is there a more appropriate English expression?